### PR TITLE
Disable TLS validation for images when IRONIC_INSECURE is True

### PR DESCRIPTION
**Pull request by dtantsur**

### CODE REVIEW

File: ironic-config/ironic.conf.j2 [hunk 1]
```diff
@@ -29,6 +29,7 @@ host = localhost
 {% else %}
 host = {{ env.IRONIC_URL_HOST }}
 {% endif %}
+webserver_verify_ca = {{ not env.IRONIC_INSECURE | bool }}
```
Comment (stbenjam) [inline ironic-config/ironic.conf.j2:32]: Could we make this configurable as a string too, to set a path to a CA?

**Comment by dtantsur** [comment]
/hold

I agree this should be configurable.

**Comment by dtantsur** [comment]
@bfournie will work on an alternative implementation

**Pull request closed without merging**